scapy/contrib/gtp_v2.py

```diff
@@ -374,15 +396,17 @@ class ULI_LAI(ULI_Field):
     fields_desc = [
         gtp.TBCDByteField("MCC", "", 2),
         gtp.TBCDByteField("MNC", "", 1),
-        ShortField("LAC", 0),
+        BitField("LAC", 0, 16),
```

Comment:
Why is this change needed? (and all of the changes related to modifying a valid Scapy field into a BitField)

Reply:
Same comment as https://github.com/secdev/scapy/pull/2488/files#r425166992 
Took the reference from existing field - https://github.com/secdev/scapy/blob/master/scapy/contrib/gtp_v2.py#L362